client/views/teams/contextualBar/info/Leave/LeaveTeamModal.js

```diff
@@ -3,59 +3,49 @@ import React, { useState, useCallback } from 'react';
 
 import { StepOne, StepTwo } from '.';
 
-export const LeaveRoomModal = ({ onCancel, onConfirm, rooms }) => {
+export const LeaveTeamModal = ({ onCancel, onConfirm, rooms }) => {
 	const [step, setStep] = useState(() => {
```

Comment:
We should try to maintain a certain pattern when using modal steps and I heavily recommend using this kind of pattern Instead of using just numbers because is more readable: 
```
const STEPS = {
	LIST_ROOMS: 'LIST_ROOMS',
	CONFIRM_DELETE: 'CONFIRM_DELETE',
};
```

Reply:
And please don't forget to add steps to reproduce section for the QA